### ISSUE TITLE
Only the first occurrence of repeating class attributes is replaced

### ISSUE THREAD

**Issue by RafaelKr** [ttskch/prettier-plugin-tailwindcss-anywhere#2]
I made a little reproduction repository:
https://github.com/RafaelKr/prettier-plugin-tailwindcss-bug

When you do
```sh
git clone https://github.com/RafaelKr/prettier-plugin-tailwindcss-bug.git
cd prettier-plugin-tailwindcss-bug
npm install

# correct output (only prettier-plugin-tailwindcss is used)
npx prettier src/demo.html

# wrong output (when @ttskch/prettier-plugin-tailwindcss-anywhere is used)
npx prettier src/demo.html.twig
```

demo.html and demo.html.twig both have the following contents:
```html
<div class="p-0 container m-0"></div>
<div class="p-0 container m-0"></div>
<div class="p-0 container m-0"></div>
```

When running prettier for the html file we receive the following output:
```html
<div class="container m-0 p-0"></div>
<div class="container m-0 p-0"></div>
<div class="container m-0 p-0"></div>
```

For the twig file the output is:
```html
<div class="container m-0 p-0"></div>
<div class="p-0 container m-0"></div>
<div class="p-0 container m-0"></div>
```

I would expect that every occurrence is replaced, independent of how often the same class attribute repeats. So the output for `.html` and `.html.twig` should not differ.

**Comment by ttskch** [comment]
Thanks for reporting the bug! I fixed it in 06e6786 and released it as [v0.0.7](https://www.npmjs.com/package/@ttskch/prettier-plugin-tailwindcss-anywhere/v/0.0.7) 🎉

**Comment by RafaelKr** [comment]
Thanks @ttskch, I can confirm it works! :tada:
